test/unit/specs/module.spec.js

```diff
@@ -19,6 +19,75 @@ describe('ti.nfc', () => {
 			it('.moduleId', () => {
 				expect(NFC.moduleId).toBe('ti.nfc');
 			});
+
+			describe('methods', () => {
+				it('nfcAdapter should be defined', () => {
+					const nfcAdapter = NFC.createNfcAdapter();
```

Comment:
Its good to create a single instance of Adapter and reuse it everywhere.

Reply:
If i am defining the same any where else, it's giving error of undefined object. That is why defined separately in each 'it' block.